### DISCUSSION TITLE
^postgres(ql)?://开头的链接怎么用

### DISCUSSION THREAD

**Discussion by stle0n** [metatube-community/jellyfin-plugin-metatube#296] [Q&A]
是否有示例呢，有点基础但不多，通常看事例能鼓捣明白，但是baidu Google没搜到这个链接格式该怎么填

**Comment by xianyunyeheA** [comment]
格式如下
postgresql://你的用户名:你的密码@你的数据库地址:你的数据库端口/你的数据库名称
例如:
postgresql://stle0n:password@stle0n.com:5432/postgres
